Mergin/metadata.txt

```diff
@@ -4,14 +4,18 @@ name=Mergin
 qgisMinimumVersion=3.22
 qgisMaximumVersion=3.99
 description=Handle Mergin Maps projects
-version=2024.2
+version=2024.2.1
 author=Lutra Consulting
 email=info@merginmaps.com
 about=Mergin Maps is a repository for storing and tracking changes to QGIS projects/data and has its mobile app for field geo-surveys. With this plugin, users can upload and sync their data to Mergin Maps service.
 ; end of mandatory metadata
 
 ; start of optional metadata
-changelog=2024.2
+changelog=2024.2.1
+  - Improve error dialog when creating and syncing projects 
```

Comment:
We usually add a ticket number (if there is one), like ` - added lorem ipsum (#879)`